Test flaky test_060_link_convergency_with_l2vpn_with_alternative_paths

The test below is being very flaky:

```
    def test_060_link_convergency_with_l2vpn_with_alternative_paths(self):
        """
        Test a simple link convergency with L2VPNs that have alternative paths:
        - Create 3 L2VPN,
        - test connectivity,
        - set one link to down,
        - wait a few seconds for convergency,
        - test connectivity again
        """
        api_url = SDX_CONTROLLER + '/l2vpn/1.0'

        payload = {"name": "Text",
                   "endpoints": [
                       {"port_id": "urn:sdx:port:ampath.net:Ampath1:50", "vlan": "100"},
                       {"port_id": "urn:sdx:port:tenet.ac.za:Tenet03:50", "vlan": "100"}
                    ]
        }
        response = requests.post(api_url, json=payload)
        assert response.status_code == 201, response.text
        h1, h8 = self.net.net.get('h1', 'h8')
        h1.cmd('ip link add link %s name vlan100 type vlan id 100' % (h1.intfNames()[0]))
        h1.cmd('ip link set up vlan100')
        h1.cmd('ip addr add 10.1.1.1/24 dev vlan100')
        h8.cmd('ip link add link %s name vlan100 type vlan id 100' % (h8.intfNames()[0]))
        h8.cmd('ip link set up vlan100')
        h8.cmd('ip addr add 10.1.1.8/24 dev vlan100')

        payload = {"name": "Text",
                   "endpoints": [
                       {"port_id": "urn:sdx:port:ampath.net:Ampath1:50", "vlan": "101"},
                       {"port_id": "urn:sdx:port:tenet.ac.za:Tenet03:50", "vlan": "101"}
                    ]
        }
        response = requests.post(api_url, json=payload)
        assert response.status_code == 201, response.text
        h1.cmd('ip link add link %s name vlan101 type vlan id 101' % (h1.intfNames()[0]))
        h1.cmd('ip link set up vlan101')
        h1.cmd('ip addr add 10.1.2.1/24 dev vlan101')
        h8.cmd('ip link add link %s name vlan101 type vlan id 101' % (h8.intfNames()[0]))
        h8.cmd('ip link set up vlan101')
        h8.cmd('ip addr add 10.1.2.8/24 dev vlan101')

        payload = {"name": "Text",
                   "endpoints": [
                       {"port_id": "urn:sdx:port:ampath.net:Ampath1:50", "vlan": "102"},
                       {"port_id": "urn:sdx:port:tenet.ac.za:Tenet03:50", "vlan": "102"}
                    ]
        }
        response = requests.post(api_url, json=payload)
        assert response.status_code == 201, response.text
        h1.cmd('ip link add link %s name vlan102 type vlan id 102' % (h1.intfNames()[0]))
        h1.cmd('ip link set up vlan102')
        h1.cmd('ip addr add 10.1.3.1/24 dev vlan102')
        h8.cmd('ip link add link %s name vlan102 type vlan id 102' % (h8.intfNames()[0]))
        h8.cmd('ip link set up vlan102')
        h8.cmd('ip addr add 10.1.3.8/24 dev vlan102')

        # test connectivity
        result_100 = h1.cmd('ping -c4 10.1.1.8')
        result_101 = h1.cmd('ping -c4 10.1.2.8')
        result_102 = h1.cmd('ping -c4 10.1.3.8')

        # set one link to down
        self.net.net.configLinkStatus('Ampath1', 'Sax01', 'down')

        # wait a few seconds for convergency
        time.sleep(15)

        # test connectivity again
        result_100_2 = h1.cmd('ping -c4 10.1.1.8')
        result_101_2 = h1.cmd('ping -c4 10.1.2.8')
        result_102_2 = h1.cmd('ping -c4 10.1.3.8')

        # clean up
        h1.cmd('ip link del vlan100')
        h1.cmd('ip link del vlan101')
        h1.cmd('ip link del vlan102')
        h8.cmd('ip link del vlan100')
        h8.cmd('ip link del vlan101')
        h8.cmd('ip link del vlan102')

>       assert ', 0% packet loss,' in result_100
E       AssertionError: assert ', 0% packet loss,' in 'PING 10.1.1.8 (10.1.1.8) 56(84) bytes of data.\r\nFrom 10.1.1.1 icmp_seq=1 Destination Host Unreachable\r\nFrom 10.1....1.1.8 ping statistics ---\r\n4 packets transmitted, 0 received, +3 errors, 100% packet loss, time 3048ms\r\npipe 4\r\n'

tests/test_05_l2vpn.py:362: AssertionError
```

--- a/tests/test_05_l2vpn.py
+++ b/tests/test_05_l2vpn.py
@@ -335,6 +335,9 @@ class TestE2EL2VPN:
         h8.cmd('ip link set up vlan102')
         h8.cmd('ip addr add 10.1.3.8/24 dev vlan102')
 
+        # wait a few seconds to allow OXPs to deploy the L2VPNs
+        time.sleep(10)
+
         # test connectivity
         result_100 = h1.cmd('ping -c4 10.1.1.8')
         result_101 = h1.cmd('ping -c4 10.1.2.8')
@@ -364,5 +367,5 @@ class TestE2EL2VPN:
         assert ', 0% packet loss,' in result_102
 
         assert ', 0% packet loss,' in result_100_2
-        #assert ', 0% packet loss,' in result_101_2
-        #assert ', 0% packet loss,' in result_102_2
+        assert ', 0% packet loss,' in result_101_2
+        assert ', 0% packet loss,' in result_102_2

